client/modules/IDE/components/FileNode.jsx

```diff
@@ -210,6 +210,20 @@ class FileNode extends React.Component {
     }
   };
 
+  handleOnKeyboardDelete = (event) => {
+    console.log(event.key);
+    if (event.key === 'Delete') {
+      const { id, parentId, name, t } = this.props;
+      const prompt = t('Common.DeleteConfirmation', { name });
+
+      if (window.confirm(prompt)) {
+        this.setState({ isDeleting: true });
+        this.props.resetSelectedFile(id);
+        setTimeout(() => this.props.deleteFile(id, parentId), 100);
+      }
+    }
+  };
+
```

Comment:
We can re-use the logic from `handleClickDelete` instead of recreating it.
```
if (event.key === 'Delete') {
  this.handleClickDelete();
}
```

Reply:
Maybe for clarity this method should be `handleKeyDown`, since the function itself handles all key pressed and not just the `Delete` ones.  Then it offloads the `Delete` ones to a delete-specific handler.